ui/src/services/utils.js

```diff
@@ -98,3 +98,11 @@ export const useRefreshEffect = (refreshAction, stopRefreshAction) => {
     };
   }, [dispatch, refreshAction, stopRefreshAction]);
 };
+
+export const sizeUnits = [
```

Comment:
I would add the `B` of byte for the labels, so: `KiB`, `MiB`, …

Also, I would add `{ label: 'B', value: '' }` as one may want to specify the size in bytes directly.